### DISCUSSION TITLE
Voice Library

### DISCUSSION THREAD

**Discussion by KyomiDev** [PenPow/SignalBot#14] [Development]
cc @PenPow for his advice

The production bot at the moment is small enough to only need @discordjs/voice ([see here](https://www.npmjs.com/package/@discordjs/voice)), which is recommended for discord.js V13.

_However_, we can scale up and setup Lavalink for production, which would increase complexity, but offer better performance.

If you can find a lavalink lib that supports djs@dev, should we implement it?

**Comment by PenPow** [comment]
Will look into a supported Lavalink Library.
